docs/utils.js

```diff
@@ -23,3 +25,7 @@ export const fakeIntentCreate = (action, doctype, options) => {
   }
   return p
 }
+
+export const t = (path) => {
+  return get(translations, path)
```

Comment:
You know my point about avoiding adding a dependency for some little needs. How about that:

```javascript
export const t = (path) => {
  return path.split('.').reduce((acc, part) => acc && acc[part], translations)
}
```

Reply:
```

Since lodash is already there, I am not

Reply:
TIL `yarn why` 🎉

Reply:
That being said, it should be added to the `dependencies` if we're using it directly (or a `devDep` if it's just for the docs).

Reply:
True, there you go :)

Reply:
So we finally added lodash as a dependency :/

Reply:
[](https://media.giphy.com/media/hppWdK8gcmzXq/giphy.gif)

Reply:
:sob:

Reply:
Well it was already there when you `yarn install`, and it was (probably) already shipped. So it doesn't actually change anything?